### PR TITLE
chronograf 0.2.0

### PR DESCRIPTION
**Pull request by toddboom**

### CODE REVIEW

**Comment by toddboom** [comment]
New recipe here! It's a new version of a product we had previously released without an official OSX build.

Let me know if you've got any questions!

**Comment by toddboom** [comment]
This seems to just be failing CI on El Capitan with no messages. Any chance we could still get this merged?

**Comment by mark-rushakoff** [comment]
[The console output for El Capitan](http://bot.brew.sh/job/Homebrew%20Binary%20Pull%20Requests/186/version=el_capitan/console) does have one error line:

> Error: No available formula for formulae require at least a URL 

A cursory Google search for that error didn't turn up much.

**Comment by dunn** [comment]
Probably a bot hiccup; I re-queued it.

**Comment by toddboom** [comment]
@dunn thanks! looks like it passed this time - anything we can do to help this get merged?

**Comment by marcuswestin** [comment]
:+1: Please merge :)

**Comment by gunnaraasen** [comment]
Looks like tests all pass. Any chance this can be merged soon?

File: chronograf.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        <key>StandardOutPath</key>
+        <string>#{var}/log/chronograf.log</string>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  def caveats; <<-EOS.undent
+    To run Chronograf manually, you can specify the configuration file on the command line:
+      chronograf -config=/usr/local/etc/chronograf.toml
```
Comment (dunn) [inline chronograf.rb]: Change this to `chronograf -config=#{etc}/chronograf.toml` since users can install Homebrew in places other than `/usr/local`.

**Comment by toddboom** [comment]
@dunn good catch - fixed!

**Comment by dunn** [comment]
Merged in a98d9fcb6bde0bd44d278341d3cb8ff52d3e2863, thanks!

**Pull request closed without merging**